images/cockroachdb/Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include ../image.mk
+
+# ====================================================================================
+# Image Build Options
+
+COCKROACHDB_IMAGE = $(BUILD_REGISTRY)/cockroachdb-$(GOARCH)
+IMAGES = $(COCKROACHDB_IMAGE)
+
+COCKROACHDB_BASE_IMAGE ?= cockroachdb/cockroach:v2.0.2
```

Comment:
What's the point in this variable?

Reply:
Good point.  i thought at first it was useful to have the cockroach upstream image version in this makefile for readability, but I think it actually hurts readability of the Dockerfile.  This version should just be put directly into the Dockerfile I think.